lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -191,7 +191,7 @@ extension DomNodeExtension on DomNode {
   }
 
   external DomNode removeChild(DomNode child);
-  external bool? get isConnected;
```

Comment:
I wonder why this was marked as nullable before? (Looks like a carry-over from dart:html [Node.isConnected](https://api.dart.dev/stable/2.13.1/dart-html/Node/isConnected.html), so I guess it's to accomodate older browsers where this property was not even defined, like IE)